data_safe_haven/external/api/azure_api.py

```diff
@@ -28,38 +29,43 @@
 from azure.mgmt.dns import DnsManagementClient
 from azure.mgmt.dns.models import RecordSet, TxtRecord
 from azure.mgmt.keyvault import KeyVaultManagementClient
-from azure.mgmt.keyvault.models import AccessPolicyEntry, Permissions
-from azure.mgmt.keyvault.models import Sku as KeyVaultSku
 from azure.mgmt.keyvault.models import (
+    AccessPolicyEntry,
+    Permissions,
     Vault,
     VaultCreateOrUpdateParameters,
     VaultProperties,
 )
+from azure.mgmt.keyvault.models import Sku as KeyVaultSku
```

Comment:
These should be in one include block

Reply:
You tell me 2143eec87 :wink:.

A little disapointed the autoformatters don't concatenate these.

Reply:
Looks like it is the isort style. I suppose separating aliased imports.

Reply:
Could change this behaviour with [combine-as-imports](https://beta.ruff.rs/docs/settings/#isort-combine-as-imports). Do you have a strong opinion?

I don't mind too much but would lean towards the default for simplicities sake.

Reply:
Fixed in 6a73875b